src/test/java/codeu/controller/ProfileServletTest.java

```diff
@@ -56,18 +64,17 @@ public void testDoPost_ExistingUser() throws IOException, ServletException {
     Mockito.when(mockRequest.getParameter("username")).thenReturn("test username");
     Mockito.when(mockRequest.getParameter("password")).thenReturn("test password");
 
-    UserStore mockUserStore = Mockito.mock(UserStore.class);
     Mockito.when(mockUserStore.isUserRegistered("test username")).thenReturn(true);
     Mockito.when(mockUserStore.getUser("test username")).thenReturn(user);
     profileServlet.setUserStore(mockUserStore);
 
-    HttpSession mockSession = Mockito.mock(HttpSession.class);
+
     Mockito.when(mockRequest.getSession()).thenReturn(mockSession);
 
     profileServlet.doPost(mockRequest, mockResponse);
 
     Mockito.verify(mockUserStore, Mockito.never()).addUser(Mockito.any(User.class));
```

Comment:
It shouldn't just be any user right? It should be the user you set up for this test that the mockUserStore is returning. So rather than using `Mockito.any()`, lets pass in the actual user that we expect, which is `user`defined on line 57

Reply:
Alright I will change it